data/scripts/eventcallbacks/player/default_onRotateItem.lua

```diff
@@ -0,0 +1,10 @@
+local ec = EventCallback
+
+ec.onRotateItem = function(self, item)
+	local newId = item:getType():getRotateTo()
+	if newId ~= 0 then
```

Comment:
This was and still is basically impossible, as item.isRotatable checks if rotateTo is not 0. I have just kept it as-it-was in C++ 🙈